tensorflow/contrib/cmake/README.md

```diff
@@ -181,7 +181,16 @@ Step-by-step Windows build
    More? -Dtensorflow_ENABLE_GPU=ON ^
    More? -DCUDNN_HOME="D:\...\cudnn"
    ```
-    
+   To enable SIMD instructions with MSVC, as AVX and SSE, define them as follows:
+   ```
+   More? -Dtensorflow_WIN_CPU_SIMD_OPTIONS=/arch:AVX
+   ```     
+   or if more than one as in:
+   ```
+   More? -Dtensorflow_WIN_CPU_SIMD_OPTIONS=/arhc:[AVX2|SSE2]
```

Comment:
"arhc" should be "arch" here :).

The syntax reads weirdly to me... `[AVX2|SSE2]` implies (to me) that you can specify `/arch:AVX2` *or* `/arch:SSE2`, but not both together. (It's weird to see a pipe symbol in the middle of a flag, for one thing....) Does it actually work to put `/arch:[AVX2|SSE2]`?